tests/lib/superset.js

```diff
@@ -0,0 +1,37 @@
+// Support comparisons where the actual value is a superset of the expected value.
+//
+// Useful in comparing pristine expected output to output that includes automatically generated cruft (IDs, etc.)
+//
+// We use this general pattern in a few places, this one is tailored to the type of results returned by couchdb-lucene.
```

Comment:
Could you add more details here to illustrate the exact behaviour of the function (perhaps with some examples) to show how the behaviour goes beyond that supported by jqUnit.assertLeftHand